### PR TITLE
add tags functionality

### PR DESCRIPTION
**Pull request by yutaro-shimizu**
Now the add_post componet supports tagging functionality.

### CODE REVIEW

**Comment by sherlockieee** [comment]
So I take over this branch and:
- Add tags to JSON post and get requests
- Display tags and cities for posts
- Fix styling for login, add posts, navbar etc.

File: frontend/src/components/Home.js [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
-import {useAuth, authFetch, login, logout} from "../auth";
-import Login from './Login';
+import { useAuth } from "../auth";
+import Login from "./Login";
 
 function Home() {
-    
-    const [info, setInfo]=useState([])
-    const [show, setShow]=useState(false)
+  const [info, setInfo] = useState([]);
```
Comment (nazaryaremko) [inline frontend/src/components/Home.js:7]: I think we are missing import React, { useState} from "react" in the beginning of this file

**Review by nazaryaremko — CHANGES_REQUESTED** [review]
Very good, I think the only thing is that I would change the tag selection from a drop-down menu to a multiple-choice kind of thing? If I have time I might try to implement this later!

**Comment by nazaryaremko** [comment]
LGTM:)

**Comment by nazaryaremko** [comment]
LGTM:)

**Pull request merged**